DomainContent/Hub/pill/scripts/swallowFXPill.js

```diff
@@ -115,7 +119,9 @@
         checkIfNearHead: function() {
             if (_this.isInactive && HMD.active) {
                 var position = Entities.getEntityProperties(_entityID, "position").position;
-                if (Vec3.distance(position, MyAvatar.getJointPosition("Head")) < (CHECK_RADIUS * MyAvatar.scale)) {
+                var pillDistance = CHECK_RADIUS * MyAvatar.scale;
+                if (Vec3.distance(position, MyAvatar.getJointPosition("Head")) < pillDistance || 
+                    Vec3.distance(position, MyAvatar.getJointPosition("Neck")) < pillDistance) {
```

Comment:
These are not new changes, ignore green https://github.com/highfidelity/hifi-content/blob/master/DomainContent/Hub/pill/scripts/swallowFXPill.js

Reply:
https://github.com/highfidelity/hifi-content/commit/d9ca4aca668f0b604d3676b00429854a788e03c5